cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaBridgeClusterTest.java

```diff
@@ -1500,7 +1500,7 @@ public void testMetricsParsingNoMetrics() {
         KafkaBridge resource = new KafkaBridgeBuilder(this.resource)
                 .build();
         KafkaBridgeCluster kbc = KafkaBridgeCluster.fromCrd(Reconciliation.DUMMY_RECONCILIATION, resource, SHARED_ENV_PROVIDER);
-        assertThat((JmxPrometheusExporterModel) (kbc.metrics()), is(nullValue()));
+        assertThat((kbc.metrics()), is(nullValue()));
```

Comment:
Nit: you should also remove the extra parentheses here.

Reply:
Addressed